src/screen/Home.tsx

```diff
@@ -57,13 +68,21 @@ export default function Home() {
             </p>
           </div>
 
-          <div className="my-5">
+          <div className="my-5 flex gap-3 justify-center items-center">
             <Link to={"session"}>
```

Comment:
need to fix free tier route, as we implement wegmi provider, we removed react-router outlet.

Reply:
solve issue for guest users, we update route conditions and manage guest users using redux.